emission/integrationTests/suggestionsys/find_destination_business.dataset.json

```diff
@@ -19,7 +19,7 @@
                 "type": "Point"
             }
         },
-        "output": ["Kettle", "North Highland Avenue", true]
```

Comment:
we don't know that the other business names will follow this format.
I checked the name for Kohl's in OSM and yelp and put them in.
There is no guarantee that yelp follows the same format for other businesses and in fact it does not for "Kettle"

> if people find alternate spellings for them while evaluating google/yelp, they should change the datasets as part of their commit.

either you do that or you actually find the names in the other services and put them in. Doesn't make sense to put in random names

Reply:
Resolved